src/App/Fossa/ArchiveUploader.hs

```diff
@@ -98,7 +99,7 @@ compressFile :: Path Abs Dir -> Path Abs Dir -> FilePath -> IO FilePath
 compressFile outputDir directory fileToTar = do
   -- Without using `fromAbsDir` for each of these directories, the conversion
   -- is incorrect. `show outputDir` gives an incorrect result even though it typechecks.
-  let finalFile = toString outputDir </> fileToTar
+  let finalFile = toString outputDir </> safeSeparators fileToTar
```

Comment:
[nit]: Can we remove the above comment while we're here?  It was stale when we switched to using `toString`